grpc-api/src/main/java/io/deephaven/grpc_api/console/ConsoleServiceGrpcImpl.java

```diff
@@ -360,23 +329,16 @@ public void closeDocument(CloseDocumentRequest request, StreamObserver<CloseDocu
     public void fetchFigure(FetchFigureRequest request, StreamObserver<FetchFigureResponse> responseObserver) {
         GrpcUtil.rpcWrapper(log, responseObserver, () -> {
             final SessionState session = sessionService.getCurrentSession();
-
-            SessionState.ExportObject<ScriptSession> exportedConsole = session.getExport(request.getConsoleId());
+            final SessionState.ExportObject<Object> figure = ticketRouter.resolve(session, request.getSourceId());
```

Comment:
since we're using this instead to fetch objects by name, make sure it throws NOT_FOUND